src/ranged.cpp

```diff
@@ -3704,6 +3705,11 @@ bool gunmode_checks_common( avatar &you, const map &m, std::vector<std::string>
                             const gun_mode &gmode )
 {
     bool result = true;
+    if( you.has_trait( trait_BRAWLER ) ) {
+        messages.push_back( string_format( _( "You refuse to fire your %s." ),
```

Comment:
This should tell *why* your character refuses to use the gun. The player may forgot that their character has this trait. Or they may not consider that trait to affect this specific item.

Reply:
Something like this? 
![image](https://user-images.githubusercontent.com/56438982/174354124-eb37fef9-a3ca-48c4-be03-f80b84bf0b0d.png)